CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to Javascript
+
+Think twice, code once.
+
+Pull requests are accepted under the discretion of Javascript maintainer(s). Improve the chances of your pull requests getting merged by following the requirements and guidelines below. For any questions, please e-mail developers@sugarcrm.com.
+
+## Requirements
+- By creating a pull request, you are agreeing to [SugarCRM Open Source Contributor terms](CONTRIBUTOR_TERMS.pdf).
+
+## Guidelines
+- Make sure your pull request contains quality code. We will certainly provide constructive feedback on works in progress but we will not merge incomplete pull requests.
+- Make sure your pull requests are properly tested and maintain (or improve) the current code coverage.
+- Make sure your pull request is fully documented.
+- Reference related Github issues within commit messages and pull request comments where appropriate.
+
+## Bugs
+- File bugs or feature requests using our [Github Issue Tracker](https://github.com/sugarcrm/javascript/issues).
```

Comment:
I think we can do the same thing that I did in https://github.com/sugarcrm/ventana/blame/master/CONTRIBUTING.md#L27 where we can just add references and put all the links in the bottom.